src/content/community/releases/_index.en.md

```diff
@@ -5,6 +5,43 @@ weight = 40
 +++
 <!-- markdownlint-disable no-duplicate-header -->
 
+## v0.15.0
+
+### New features
+
+* Submariner can now be installed on a dual-stack Kubernetes cluster. Currently, it does not support IPv6, but connectivity and
+  service-discovery would continue to work normally using IPv4 addresses.
+* Added a `subctl recover-broker-info` command to recover lost `broker-info.subm` file.
+* It is now possible to customize the default TCP MSS clamping value set by Submariner also for non-Globalnet deployments.
+* The `subctl gather` command will now gather iptables logs for Calico and kindnet CNIs.
+* The `subctl gather` command now collects the `ipset` info from all the cluster nodes.
+* The `subctl diagnose` command now validates the Calico IPPool configuration to match with Submariner requirements.
+* The following deprecated commands and variants have been removed:
+  * `subctl benchmark`’s `--kubecontexts` option (use `--context` and `--tocontext` instead)
+  * `subctl benchmark`’s `--intra-cluster` option (specify a single context to run intra-cluster benchmarks)
+  * `subctl benchmark` with two `kubeconfigs` as command-line arguments
+  * `subctl cloud`’s `--metrics-ports` option
+  * `subctl deploy-broker`’s `--broker-namespace` option (use `--namespace` instead)
+  * `subctl diagnose firewall metrics` (this is checked during deployment)
+  * `subctl diagnose firewall intra-cluster` with two `kubeconfigs` as command-line arguments
+  * `subctl diagnose firewall inter-cluster` with two `kubeconfigs` as command-line arguments
+  * `subctl gather`’s `--kubecontexts` option (use `--contexts` instead)
+
+### Other changes
+
+* The namespace is now included in `EndpointSlice` names to avoid conflicts between services with the same name in multiple namespaces.
+* Fix issues in Azure cloud prepare:
```

Comment:
```suggestion
* Fixed issues in Azure cloud prepare:
```

Reply:
I see `Fixed` is some places. Past tense seems more natural to me. Present tense connotes an instruction to do so. Also L14 "_Added a ..._" is past tense. So to be consistent then that should be present tense but saying "_Add a ..._" doesn't make sense.

Reply:
I changed it to "Add a" (makes sense to me?).

I think we mostly use present tense. [Wikipedia says present tense](https://en.wikipedia.org/wiki/Release_notes#:~:text=Release%20notes%20are%20frequently%20written,clear%2C%20correct%2C%20and%20complete.), [K8s seems to use a mix](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.9.md) (because it's auto-gen'd from commit messages I think).

Reply:
Wikipedia says "Release notes are **frequently** written in the present tense..." but I don't think that necessarily means they **should** be. I don't want to belabor it but I still think past tense makes more sense but if the consensus is otherwise then I'm OK with it. @skitt @sridhargaddam @nyechiel

Reply:
Even I feel using past tense makes more sense.

Reply:
https://github.com/submariner-io/submariner-website/pull/930